packages/manager/apps/web/client/app/private-database/dashboard/translations/Messages_de_DE.json

```diff
@@ -470,7 +469,7 @@
   "privateDatabase_modale_oom_tab_size_reached": "Erreichte Kapazität",
   "privateDatabase_modale_oom_tab_uptime": "Zeit seit der letzten Überschreitung",
   "privateDatabase_modale_oom_no_oom": "Keine Überschreitung der RAM-Kapazität.",
-  "privateDatabase_modale_oom_info_order": "Wir empfehlen Ihnen, die RAM-Kapazität Ihrer {{t0}} zu erhöhen, um diese Probleme zu beheben.",
+  "privateDatabase_modale_oom_info_order": "Wir empfehlen Ihnen, die RAM-Kapazität Ihrer CloudDB zu erhöhen, um diese Probleme zu beheben.",
```

Comment:
According to https://github.com/ovh/manager/blob/master/CONTRIBUTING.md#translations only `Messages_fr_FR.json` should be updated

Reply:
Seen together, not critical for this one.